brainfuck/parser_test.go

```diff
@@ -0,0 +1,44 @@
+package brainfuck
+
+import (
+	"reflect"
+	"testing"
+)
+
+func TestParser_parse(t *testing.T) {
+	type args struct {
+		input string
+	}
+	tests := []struct {
+		name     string
+		args     args
+		expected []command
+	}{
+		{
+			name: "should parse proper input",
+			args: args{"[+-><.]"},
+			expected: []command{
+				loop{
+					[]command{
+						increment{}, decrement{}, moveForward{}, moveBackward{}, output{},
+					},
+				},
+			},
+		},
+		{
+			name:     "should parse input with third party symbols",
```

Comment:
I would rename this test to something like this: "parse should skip third party symbols"